Enterprise-CMCS/macpro-onemac
# Oy2 30795

Story: https://jiraent.cms.gov/browse/OY2-30795
Endpoint: https://d1b2b4p08nc497.cloudfront.net/

### Details

Purpose of the card is to fix white screen after prompt on dashboard, if user clicks outside the prompt box then clicks "FAQ" in the navigation bar the current behavior in prod is a blank white screen

Added fix for auto logout after one hour

### Changes

When A user clicks "Continue Browsing" on the prompt, the user gets one more hour session time. 

### Implementation Notes

We May want to consider in the future adding refresh token functionality. 

### Test Plan
Scenario 1)
1.  Login and wait 45 minutes
2. When prompted with logout/keep browsing prompt, click outside the prompt 
3.  Click FAQ
4. See  that Screen does not blank out.

Scenario 2)
1.  Login and wait 45 minutes
2. When prompted with logout/keep browsing prompt, click "Continue Browsing"
3. Verify that after one hour the user is not logged out automatically

Scenario 3)
1.  Login and wait 45 minutes
2. When prompted with logout/keep browsing prompt, click "Logout"
3.  User is Logged out

scenario 4)
1. Login and wait 1 hour & ignore prompt
2. User automatically logs out after one hour
